types/lhr/lhr.d.ts

```diff
@@ -143,4 +146,13 @@ declare module Result {
   }
 }
 
+/**
+ * The full output of a Lighthouse flow. Includes a series of Lighthouse runs.
+ * TODO(FR-COMPAT): Add flow specific metadata (e.g. interaction steps).
+ */
+export interface FlowResult {
```

Comment:
`FlowResult` sounds good to me, but what do you think about a new `flow.d.ts` in this directory?